src/dbnode/persist/fs/commitlog/commit_log_test.go

```diff
@@ -134,16 +137,31 @@ type testWrite struct {
 }
 
 func testSeries(
+	t *testing.T,
+	opts Options,
 	uniqueIndex uint64,
 	id string,
 	tags ident.Tags,
 	shard uint32,
 ) ts.Series {
+	var (
+		tagEncoderPool = opts.FilesystemOptions().TagEncoderPool()
+		tagSliceIter   = ident.NewTagsIterator(ident.Tags{})
```

Comment:
nit: we've had problems with initing default pools in tests which end up being huge, especially when they're created in test functions... instead, we can create it in an init() function with a set size?

Reply:
Ah nice, it comes from opts. disregard